### ISSUE TITLE
Trained the model with original codes and settings, but cannot obtain the results in the paper.

### ISSUE THREAD

**Issue by Infinitywxh** [ChuhuaW/SGNet.pytorch#6]
I trained the model with train_cvae.py and evaluate with eval_cvae.py, and didn't change any settings in the original codes, but there's large gap between the results and the performance reported in the paper. Are there any additional tricks for the training process not mentioned? And I wonder if you could release the pretrained model with the reported performance? Thanks!

**Comment by ChuhuaW** [comment]
Thanks for your interest in our paper! Which dataset did you use?

**Comment by Infinitywxh** [comment]
The ETH-UCY dataset, including eth, univ, hotel, zara1, zara2.

**Comment by ChuhuaW** [comment]
Have you followed trajectron++ to preprocess the data? Also, for each dataset in ETHUCY, maybe you can try different dropout rate. We use 0.5, 0.3, 0.0, 0.0, 0.0 for eth, hotel, univ, zara1, zara2 respectively.  For eth, you may also try set the sigma to 1.5. We will release the checkpoints as well as the codes and results for NuScenes dataset in the next update. Stay tuned!

**Comment by Infinitywxh** [comment]
Thanks for your prompt reply! I followed Trajectron++ preprocessing techniques. I'll try the dropout rate and sigma you mentioned and see if the results can be better.
